packages/xrpl/test/integration/requests/accountInfo.test.ts

```diff
@@ -80,4 +80,105 @@ describe('account_info', function () {
     },
     TIMEOUT,
   )
+
+  it(
+    'uses api_version 1',
```

Comment:
regarding the integration tests that mention `api_version 1`, how are we enforcing that? -- the present CI/CD defaults to v2.

henceforth, are we executing integration tests against two versions of rippled? v1 and v2? I suspect we'll need to make changes to the Github Actions workflow right?

Reply:
It doesn't default to API v2. The client library integ tests decide which api_version to use. I don't think it's necessary to have separate GitHub Action workflows since not all requests were affected by API v2. For the ones that were, I kept their API v1 tests and added v2.

Reply:
ah okay, I didn't realise that.

The xrpl docs do not mention the `api_version` parameter, so I was slightly confused -- https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/account-methods/account_info/

Reply:
api_version can be included with any request, so it's mentioned here: https://xrpl.org/docs/references/http-websocket-apis/api-conventions/request-formatting/

Reply:
thanks @intelliot 

there's an interesting discrepancy here -- rippled defaults to API v1 whereas the xrpl.js client library defaults to using V2. I hope developers don't get confused by that. It shouldn't be much of a problem because there are no V1 machines (afaik)

Reply:
@ckeshava that's a good callout about being a discrepancy between rippled and client libraries but I still agree to default to API v2. And I would assume eventually rippled will default to v2 too.